src/android/Print.java

```diff
@@ -722,16 +724,31 @@ private int getPrinterSeries(String target) {
 		return series;
 	}
 
+	// As of version code 95 (version name 1.2.95), the PrintConnect app must be triggered in the foreground
+	private static final long NEW_PRINT_CONNECT_VERSION = 95;
```

Comment:
I think this code will break when version 1.3.0 is released

Rather than handling that case via code, it might be easier to have a follow up PR for once GPM is all on the newest version of printconnect. After they're all updated, we can just remove all the version checking stuff and always do the foreground approach

Reply:
That said, a follow up PR to simplify the logic once more after the tablets are fully updated would be reasonable. That may involve some amount of coordination with GPM to ensure they are up-to-date.